docs/tutorials/visibility_simulator.ipynb

```diff
@@ -57,12 +57,22 @@
   },
```

Comment:
<p>I have code somewhere for gridding point source catalogs and making them into a nice heatmap for plotting. Lemme know if we want to update this plot to use that code (I'm having trouble attaching an example plot, but I can provide one in a comment on the PR if requested).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I think this is fine for now (the heatmap is nice but I think here a scatter plot is perfectly fine)</p>

Reply:
<p>Sounds good</p>